control_fun.h

```diff
@@ -273,13 +275,17 @@ class PIDController {
     void set_param(const PIDParam &param);
     float get_error() const { return error_last_; }
     void set_rollover(float rollover) { rollover_ = rollover; }
+    bool is_saturated() const {
+        return std::abs(filtered_out_) >= command_max_;
```

Comment:
This should really have an epsilon in here, basically >= command_max_ - eps. Would you agree?